.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Existing functionality does not work as intended
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe The Bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Set up a repository with the following structure '...'
+2. Run `sap-cloud-sdk ...` 
+3. Confirm prompt '....'
+4. See error
+
+**Expected Behavior**
+A clear and concise description of what you expected to happen.
```

Comment:
"...of what you expect to happen." or even better "... of the expected behavior."